### PR TITLE
Custom Error Page for 403, 404 and 500

### LINKED ISSUE #111: Redirect to new 404 and 403 page in case of these errors
We have debug turned on develop, should we turn it off there? 

Also on the main one I tried access a 404 and got a screen like this. User should be redirected to a specific 404 informative page and after we merge user registration, login we also need 403 page. 

[Screenshot from Gyazo](https://gyazo.com/7f7e5dc9e013d1a1c169a9669925eef7)

### PR DESCRIPTION
**Pull request by bhatiatanish**
**Checklist**

- [x] My branch is up-to-date with the upstream `develop` branch.
- [x] I have added necessary documentation (if appropriate).

**Which issue does this PR fix?**: fixes #111 

Closes #111 will handle redirect to custom 403, 404 and 500 error page that blend with the original theme of website.

**Why do we need this PR?**:

To handle Error 403, 404 and 500 via custom webpage redirect.

Like you mentioned it! xD

### CODE REVIEW

**Comment by vaibhavsingh97** [comment]
@bhatiatanish wow!! You have done an awesome job. Since this is your first PR and you have written so much code thing so clap for your self.  👏 👏  We need contributors like you so keep doing this 😄 
Hound CI had commented some issues with the PR, can you please try to fix it. It's ok of you don't know and you can ask for help anytime, we are always available to help devs like you

**Comment by vaibhavsingh97** [comment]
@bhatiatanish  Also, it's better that after solving the issues you please add screenshot in the comment box so that reviewer like us could get idea about what you have changed. Also, write `Fixes` ,`closes` etc words so after merge this will close the issue automatically. You can read more about it [here](https://help.github.com/articles/closing-issues-using-keywords/)

**Comment by bhatiatanish** [comment]
@vaibhavsingh97 Thank you for the appreciation I'll definitely look into the issues marked by Hound Cl.

**Comment by nikhita** [comment]
@bhatiatanish Can you add a screenshot like @vaibhavsingh97 mentioned? Thanks!

**Comment by vaibhavsingh97** [comment]
@bhatiatanish I am waiting for your screenshot before i start my review 😅

File: oshc/main/templates/403.html
Comment (tapaswenipathak) [inline oshc/main/templates/403.html:10]: this won't show the url down when user hovers over go back button.

File: oshc/main/templates/403.html
Comment (tapaswenipathak) [inline oshc/main/templates/403.html:8]: r/don;t/don't.

**Review by tapaswenipathak — APPROVED** [review]
I tried few, 404 looks like 
test404

similarly other two. To test other two you need to raise exception.

Thanks for the work @bhatiatanish! I will do few changes in this, squash commits, merge in my local and then push to develop. :rocket:

File: oshc/main/templates/404.html
Comment (tapaswenipathak) [inline oshc/main/templates/404.html:8]: r/Sorry!/Woops! 😄

**Comment by bhatiatanish** [comment]
@tapasweni-pathak and @vaibhavsingh97 I was quite busy and couldn't check for any tags. Quite new to this but will try to keep this in mind. Again Sorry !!

**Comment by vaibhavsingh97** [comment]
@bhatiatanish No need to sorry. It's a collaborative learning process sometime you learn with us and sometimes we learn from you. Keep in mind no one is 100% perfect. 😄

**Pull request closed without merging**
